test/language/statements/class/elements/privatemethods-on-proxy.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2019 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Successfully access private method on Proxy objects
+esid: sec-privatefieldget
+info: |
+  PrivateFieldGet(P, O)
+    ...
+    4. Perform ? PrivateBrandCheck(O, P).
+    5. If P.[[Kind]] is "method",
+      a. Return P.[[Value]].
+      ...
+includes: [compareArray.js]
+features: [class, class-fields-private, Proxy]
```

Comment:
`class-methods-private` instead of class-fields-private

Reply:
Oops.